### PR TITLE
Add direction control for i18n

### PR DESCRIPTION
**Pull request by Artoria2e5**
Currently the farsi and arabic pages are full of explicit directions.

Doesn't need to be like that.

(I will also get a PR out to actually use these things. That one would depend on this one for obvious reasons.)

### CODE REVIEW

**Comment by ehashman** [comment]
Compare to #2, this is a bit more convoluted. Can we just check the languages directly and control like that?

**Comment by Artoria2e5** [comment]
Pretty sure we can. Although I would rephrase it like:

```liquid
{% assign rtl_langs = "ar, arc, dv, fa, ha, he, khw, ks, ku, ps, ur, yi" | split: ", " %}
<!-- ... -->
  {%- if rtl_langs contains page.lang -%}  
  <div class="post-content" dir="rtl">
  {%- else -%}
  <div class="post-content">
  {%- endif -%}
```

**Comment by ehashman** [comment]
I'd support the rephrase, I was just doing an MVP :)

**Comment by Artoria2e5** [comment]
Let me use that instead then. And also simplify the PR that depends on it.
